plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -248,8 +248,10 @@ public DistributedQueryRunner build()
                         .put("hive.max-initial-split-size", "10kB") // so that each bucket has multiple splits
                         .put("hive.max-split-size", "10kB") // so that each bucket has multiple splits
                         .put("hive.storage-format", "TEXTFILE") // so that there's no minimum split size for the file
-                        .put("hive.compression-codec", "NONE") // so that the file is splittable
                         .buildOrThrow();
+                hiveBucketedProperties = new HashMap<>(hiveBucketedProperties);
```

Comment:
undo?

Reply:
it's intentional. i need to override compression for the second catalog gets registered here